src/components/Header/Header.module.scss

```diff
@@ -286,3 +286,11 @@ $header-transition: all 0.3s;
     transform: translateY(0);
   }
 }
+
+:global {
+  .boxshadow-out > div {
```

Comment:
instead of using global styles, can you pass state to the context and get it in the header component? 

`Layout.js`:
```
<BoxShadowContext.Provider value={{ removeBoxShadow,  remBoxShadow}}>
```

and move this code 
```
const headerClassName = classNames({
     'boxshadow-out': remBoxShadow,
   });
```

to the `Header.js`?